message StrokeInfo {
  message Tile {
    required sint32 tx = 1;
    required sint32 ty = 2;
    optional bytes data_compressed = 3;
  }
  repeated Tile tiles = 2;
  optional bytes brush_string_compressed = 3;
}

message StrokeList {
  repeated StrokeInfo strokes = 1;
}
